src/main/kotlin/minesweeper/model/Cell.kt

```diff
@@ -1,14 +1,24 @@
 package minesweeper.model
 
-@JvmInline
-value class Cell private constructor(
+data class Cell(
     private val type: CellType,
+    val position: CellPosition,
 ) {
+
+    fun findSurroundingMineCountSum(board: MineBoard): Int {
+        val surroundingPositions = position.findSurroundingCellPositions()
+        return board.board.sumOf {
+            it.count { cells -> cells.isMineIn(surroundingPositions) }
```

Comment:
cells가 아니라 cell이 맞는 것 같아요!

```suggestion
            it.count { cell -> cell.isMineIn(surroundingPositions) }
```

Reply:
매의 눈을 가지셨군요..!! 꼼꼼한 피드백 감사드려요~ 👍